pkg/graph/schema.resolvers.go

```diff
@@ -19,6 +20,44 @@ func (r *accessibilityRequestResolver) Documents(ctx context.Context, obj *model
 	panic(fmt.Errorf("not implemented"))
 }
 
+func (r *accessibilityRequestResolver) RelevantTestDate(ctx context.Context, obj *models.AccessibilityRequest) (*model.TestDate, error) {
+	allDates := []*model.TestDate{}
```

Comment:
Is it worth doing an early return if `allDates` is empty?

Reply:
Right now, this is just a placeholder for actually talking to the data layer to fetch all the dates, which I was planning on doing in a future PR... But can try to add in this PR if you want.